shared/xccdf/services/ssh.xml

```diff
@@ -49,6 +49,16 @@ operator="equals" interactive="0">
 <value selector="default">hmac-sha2-512,hmac-sha2-256,hmac-sha1,hmac-sha1-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com</value>
 </Value>
 
+<Value id="sshd_required" type="number"
+operator="equals" interactive="0">
+<title>SSH is required to be installed</title>
+<description>Specify if the Policy requires SSH to be installed. Used by SSH Rules
+to determine if SSH should be uninstalled or configured.</description>
```

Comment:
Can we describe here which value does what? In SCAP Workbench, there is no information whatsoever.